goodai/ltm/memory.py

```diff
@@ -125,6 +125,7 @@ class RealTimeLTMSystem:
     def __init__(
         self, embedding_model: RemoteEmbeddingModel = None,
         background_process_fn: Callable[[dict], dict] = None, time_budget: float = 1,
+        embedding_model_name: str = None
```

Comment:
This way is a bit confusing at first, but I agree that it's a good fallback for simplifying the instantiation. We should revisit this in the future to avoid having redundant configuration elements.

Reply:
I was trying to come up with a good way to keep this reminder... maybe a `TODO` comment?